docs/source/index.rst

```diff
@@ -47,39 +47,41 @@ With Docker installed:
 
 ``docker compose up --pull=always``
 
-5.	The Sycamore demo query UI will be at localhost:3000
+5.	The Sycamore demo query UI is located at: 
 
-You can next choose to run a demo that [prepares and ingests data from the Sort Benchmark website](docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-sort-benchmark-dataset), [crawl data from a public website](docs/source/welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website), or write your own data preparation script.
+``http://localhost:3000/``
 
-For more info about Sycamore’s data ingestion and preparation feature set, visit the [Sycamore documentation](docs/source/data_ingestion_and_preparation/data_preparation_concepts.md).
+You can next choose to run a demo that :doc:`prepares and ingests data from the Sort Benchmark website </welcome_to_sycamore/get_started.md#demo-ingest-and-query-sort-benchmark-dataset>`, :doc:`crawl data from a public website <welcome_to_sycamore/get_started.md#demo-ingest-and-query-data-from-an-arbitrary-website>`, or :doc: `write your own data preparation script </data_ingestion_and_preparation/using_jupyter>`.
```

Comment:
Where is the :doc: thing coming from? I don't see it in the github markdown documentation.
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links
I expected the old `[text](link)` form.

Reply:
This is a rst file, so needed different syntax for links.